starter_code/views/albums.hbs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <ul>
+        {{#albums}}
+        <li> <a href="/tracks/{{this.id}}">{{ this.name }}</a> </li> {{/albums}} </ul>
```

Comment:
To display the images something like that:
`    {{#each artistAlbums}}
    <div class="album-box">
      <li>
        <img src="{{this.images.1.url}}" alt="" />
        <p>{{this.name}}</p>
        <button><a href="/tracks/{{this.id}}">View Tracks</a></button>
      </li>
    </div>
    {{/each}}`